### DISCUSSION TITLE
Not loading models

### DISCUSSION THREAD

**Discussion by patriciapreda** [google/model-viewer#3162] [Q&A]
Hello, I am facing quite a strange issue. I am trying to upload a model from my computer using 'model-viewer' but it only shows the progress bar and then white like nothing was loaded. When I try the exact same code but I change the src to 'https://modelviewer.dev/shared-assets/models/NeilArmstrong.glb' everything works and the model loads. 

What I have tried to do:

1. Download the model Neil Armstrong and upload it from my computer (to check if my model might be wrong) but still doesn't work
2. Upload my model onto a website I host and tried to src as: https://website/model.glb and still no luck
3. I have also tried 'Download scene' and if I delete the poster part from code it shows the progress bar and then white

Here is the code I am using (taken from https://developers.google.com/ar/develop/webxr/model-viewer):

```
<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>
  <title>Document</title>

  <style>
    .box {display: flex;}
    model-viewer {width: 800px; height: 600px; margin: 0, auto;}
  </style>
</head>

<body>
  <div class="box">
    <div><p>TEXT</p></div>
    <!--<div><model-viewer src="assets/images/sm.glb"></model-viewer>-->
    <model-viewer src="https://modelviewer.dev/shared-assets/models/Astronaut.glb"
              alt="A 3D model of an astronaut"
              ar
              auto-rotate
              camera-controls></model-viewer>
    </div>
  </div>
</body>
</html>
```

**Comment by samaneh-kazemi** [comment]
Hi @patriciapreda , thanks for reaching out. Do you mind sharing a link to your code(rather than raw code) so we can try it on our end? You can try using [glitch](https://glitch.com/).

**Reply by patriciapreda** [reply]
Hello, here you can find the 
preview: https://handy-wide-pillow.glitch.me
and the code: https://glitch.com/edit/#!/handy-wide-pillow

Thank you!

**Reply by elalish** [reply]
@patriciapreda I just tried your Glitch link and it all seems to be working now?

**Reply by patriciapreda** [reply]
Yes, it does, my model might have been too big so I have resized it using gltf pipeline. Thank you!

**Comment by samaneh-kazemi** [comment]
Can you please share your model here as well? Also, have you tried loading your model in editor? The link you shared earlier is not private so I can't open it: https://website/model.glb

**Comment by OmerFainshtein** [comment]
hey,
i have the same problem.
How can I reach you for some questions?
